src/lib.rs

```diff
@@ -648,6 +648,33 @@ impl Builder {
         self
     }
 
+    /// Add multiple input C/C++ headers.
+    pub fn headers<T: AsRef<str>>(
+        mut self,
+        headers: impl IntoIterator<Item = T>,
+    ) -> Builder {
+        for path in headers {
+            self = self.header(path.as_ref());
+        }
+        self
+    }
+
+    /// Include a folder for system header file resolution
+    pub fn include(self, path: impl AsRef<str>) -> Builder {
+        self.clang_arg(format!("-I{}", path.as_ref()))
+    }
+
+    /// Include multiple folders for system header file resolution
+    pub fn includes<T: AsRef<str>>(
```

Comment:
Personally I think having both `includes` and `include` is probably not necessary, except for consistency with `headers` and `header`; if we were starting from scratch, I would prefer just to have the plural versions, and call them with an array of size one, or `std::iter::once`, if I have only one element. But maybe that is just me.

But removing `header` would break backward compatibility, and having both `header` and `headers` but only `includes` seems inconsistent, so maybe I do not have anything better to suggest here.

Reply:
`include_directories`?